modules/i18n/input-method/fcitx5.nix

```
@@ -18,6 +18,14 @@ in {
           Enabled Fcitx5 addons.
         '';
       };
+      waylandFrontend = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc ''
```

No need for `mdDoc`, it is the default anyway

```suggestion
        description = ''
```